takwimu/takwimu_ui/src/components/Navigation/DataByTopic.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { withStyles, Grid, Typography, ButtonBase } from '@material-ui/core';
+
+import chartBackground from '../../assets/images/a-chart-bg.svg';
+
+const flagSrc = require.context('../../assets/images/flags', false, /\.svg$/);
```

Comment:
QQ: Do these get the same benefits of normal imports i.e. [Data URI](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs), compile errors instead of runtime 404s, etc.

Reply:
this will not have a runtime error unless we manually load each image and add it to a dictionary, the only other option i can think of right now.